app/questionnaire/path_finder.py

```diff
@@ -198,8 +200,49 @@ def get_next_location(self, current_location):
         current_location_index = PathFinder._get_current_location_index(routing_path, current_location)
 
         if current_location_index is not None and current_location_index < len(routing_path) - 1:
+            completed_blocks = get_completed_blocks(current_user)
+            if self._is_survey_completed(routing_path, completed_blocks):
+                return routing_path[-1]     # Go to Summary location
```

Comment:
I don't mind doing that but i'm not sure how more readable that is. 
@nealedj, @shahi645  thumbs up for change, thumbs down for not required please.
It would never be the same as the declaration of variable `section_summary_location` as that is set by a method. That method could then return None and if so I wouldn't return `section_summary_location` at all. Here we would always return the summary location so it's just creating a variable to immediately return it.

Reply:
I'll say no in this case. Although assigning values to variables in order to help add context is a valid point.

It's a reasonable implicit bit of logic that if the survey is complete then the next location must be the last (final) location.